test/data/testinit.js

```diff
@@ -346,16 +346,16 @@ this.includesModule = function( moduleName ) {
 
 	var excludedModulesPart, excludedModules;
 
-	// A short-cut for the slim build, e.g. "4.0.0-pre slim"
-	if ( jQuery.fn.jquery.indexOf( " slim" ) > -1 ) {
+	// A short-cut for the slim build, e.g. "4.0.0-pre+slim"
+	if ( jQuery.fn.jquery.indexOf( "+slim" ) > -1 ) {
```

Comment:
Actually, this whole block is not needed. In `3.x`, we don't use the `slim` shortcut, we just list all the equivalent exclusions.